workflow/rules/samtools_sort.smk

```diff
@@ -0,0 +1,37 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Patrik Smeds"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "patrik.smeds@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule samtools_sort:
+    input:
+        "{path_file}.bam_unsorted",
+    output:
+        "{path_file}.bam",
+    log:
+        "{path_file}.samtools_sort.log",
```

Comment:
It seems that we are using different systems for naming the log and benchmark files. We should agree on one way I think. Let's discuss soon.